stabilize.md

```diff
@@ -56,21 +66,130 @@ Flap Semantics
     syntax AuthStep ::= FlapContract "." FlapAuthStep [klabel(flapStep)]
  // --------------------------------------------------------------------
     rule <k> Flap . _ => exception ... </k> [owise]
+```
 
-    syntax FlapStep ::= "kick" Rad Int
- // ----------------------------------
+- kick(uint lot, uint bid) returns (uint id)
+- Starts a new surplus auction for a lot amount
 
-    syntax FlapStep ::= "tend" Int Int Int
+```k
+    syntax FlapAuthStep ::= "kick" Rad Wad
  // --------------------------------------
+    rule <k> Flap . kick LOT BID
+          => call Vat . move MSGSENDER THIS LOT
+          ~> KICKS +Int 1
+          ...
+         </k>
+         <msg-sender> MSGSENDER </msg-sender>
+         <this> THIS </this>
+         <currentTime> NOW </currentTime>
+         <flap-bids>... .Map =>
+            KICKS +Int 1 |-> FlapBid(... bid: BID,
+                                         lot: LOT,
+                                         guy: MSGSENDER,
+                                         tic: 0,
+                                         end: NOW +Int TAU)
+         ...</flap-bids>
+         <flap-kicks> KICKS => KICKS +Int 1 </flap-kicks>
+         <flap-live> true </flap-live>
+         <flap-tau> TAU </flap-tau>
+```
 
-    syntax FlapStep ::= "deal" Int
- // ------------------------------
+- tick(uint id)
+- Extends the end time of the auction if no one has bid yet
 
-    syntax FlapAuthStep ::= "cage" Int
+```k
+    syntax FlapStep ::= "tick" Int [klabel(FlapTick),symbol]
+ // --------------------------------------------------------
+    rule <k> Flap . tick ID => . ... </k>
+         <currentTime> NOW </currentTime>
+         <flap-bids>...
+           ID |-> FlapBid(... tic: 0, end: END => NOW +Int TAU)
+         ...</flap-bids>
+         <flap-tau> TAU </flap-tau>
+      requires END <Int NOW
+```
+
+- tend(uint id, uint lot, uint bid)
+- Places a bid made by the user. Refunds the previous bidder's bid.
+
+```k
+    syntax FlapStep ::= "tend" Int Rad Wad
+ // --------------------------------------
+    rule <k> Flap . tend ID LOT BID
+          => call Gem "MKR" . move MSGSENDER GUY BID'
+          ~> call Gem "MKR" . move MSGSENDER THIS (BID -Int BID')
+         ...
+         </k>
+         <msg-sender> MSGSENDER </msg-sender>
+         <this> THIS </this>
+         <currentTime> NOW </currentTime>
+         <flap-bids>...
+           ID |-> FlapBid(... bid: BID' => BID,
+                              lot: LOT',
+                              guy: GUY => MSGSENDER,
+                              tic: TIC => TIC +Int TTL,
+                              end: END)
+         ...</flap-bids>
+         <flap-live> true </flap-live>
+         <flap-ttl> TTL </flap-ttl>
+         <flap-beg> BEG </flap-beg>
+      requires GUY =/=Int 0
```

Comment:
This should be `=/=K`, not `=/=Int`, since `guy` is an `Address`, which could be an `Int` or a `String`.

Reply:
I will just take the requires statement out, since it came from an ethereum specific check.